pcrelate.R

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+
+st_help <- "Threshold for sparsifying GRM (will be multiplied by scale_kin)"
```

Comment:
same comment as above: code is easier to read if this string is right next to the argument